rpcserver.go

```diff
@@ -7799,60 +7797,60 @@ const (
 
 // rfqChannel returns the channel to use for RFQ operations. If a peer public
 // key is specified, the channels are filtered by that peer. If there are
-// multiple channels for the same asset, the user must specify the peer public
-// key.
-func (r *rpcServer) rfqChannel(ctx context.Context, id asset.ID,
+// multiple channels for the same specifier, the user must specify the peer
+// public key.
+func (r *rpcServer) rfqChannel(ctx context.Context, specifier asset.Specifier,
 	peerPubKey *route.Vertex,
-	intention chanIntention) (*channelWithAsset, error) {
+	intention chanIntention) (*channelWithSpecifier, error) {
 
-	balances, err := r.computeChannelAssetBalance(ctx)
+	balances, err := r.computeChannelAssetBalance(ctx, specifier)
 	if err != nil {
 		return nil, fmt.Errorf("error computing available asset "+
 			"channel balance: %w", err)
 	}
 
-	assetBalances, haveBalance := balances[id]
-	if !haveBalance || len(assetBalances) == 0 {
-		return nil, fmt.Errorf("no asset channel balance found for "+
-			"asset %s", id.String())
+	if len(balances) == 0 {
+		return nil, fmt.Errorf("no asset channel balance found for %s",
+			specifier.String())
```

Comment:
I think because `.String()` has a pointer receiver, it's enough to just do `&specifier`.

Reply:
Can be applied to the other error messages below as well (`%v` and pointer).